package.json

```diff
@@ -14,6 +14,8 @@
     "eslintfix": "eslint \"**/*.{js,vue}\" --fix",
     "i18n": "node scripts/i18n.js",
     "flow": "flow",
+    "flow-typed:install": "flow-typed install --ignoreDeps bundled peer --libdefDir ./shared/libdefs",
```

Comment:
Looks good. Once this is extracted into a `postinstall` hook, I can test it and merge it.

I'm not too concerned about the repetition of arguments at this point, since there's only going to be three instances of it in this file (these two and the new one), and it's not likely to change very often.

But if you're feeling up to it, feel free to extract a bash command that runs `flow-typed ${cmd} --ignoreDeps bundled peer --libdefDir ./shared/libdefs` and can be invoked as `some-cmd install` in `flow-typed:install` and `postinstall`, and `some-cmd update` in `flow-typed:update`.

Or better yet, if we find that the `postinstall` hook is sufficient for most cases, and if `npm update` runs it, we can just leave the one instance there.

Reply:
That makes sense.
Besides, given that the project uses only exact semver versions for its dependencies, for me it looks like `npm update` is never used.
If that's correct, then the postinstall script will likely be enough.